examples/tree_1d_dgsem/elixir_advection_diffusion.jl

```diff
@@ -0,0 +1,86 @@
+
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# semidiscretization of the linear advection equation
+
+advection_velocity = 1.0e-1
+equations = LinearScalarAdvectionEquation1D(advection_velocity)
+diffusivity() = 1.0e-1
+equations_parabolic = LaplaceDiffusion1D(diffusivity(), equations)
+
+# Create DG solver with polynomial degree = 3 and (local) Lax-Friedrichs/Rusanov flux as surface flux
+solver = DGSEM(polydeg=3, surface_flux=flux_lax_friedrichs)
+
+coordinates_min = -pi # minimum coordinate
+coordinates_max =  pi # maximum coordinate
+
+# Create a uniformly refined mesh with periodic boundaries
+mesh = TreeMesh(coordinates_min, coordinates_max,
+                initial_refinement_level=7,
+                n_cells_max=30_000) # set maximum capacity of tree data structure
+
+# Define initial condition
+function initial_condition_diffusive_convergence_test(x, t, equation::LinearScalarAdvectionEquation1D)
+    # Store translated coordinate for easy use of exact solution
+    x_trans = x - equation.advection_velocity * t
+  
+    nu = diffusivity()
+    c = 0.0
+    A = 1.0
+    L = 2
+    f = 1/L
+    omega = 1.0
+    scalar = c + A * sin(omega * sum(x_trans)) * exp(-nu * omega^2 * t)
+    return SVector(scalar)
+  end
+  initial_condition = initial_condition_diffusive_convergence_test
+  
+  # define periodic boundary conditions everywhere
+  boundary_conditions = boundary_condition_periodic
+  boundary_conditions_parabolic = boundary_condition_periodic
```

Comment:
```suggestion
function initial_condition_diffusive_convergence_test(x, t, equation::LinearScalarAdvectionEquation1D)
  # Store translated coordinate for easy use of exact solution
  x_trans = x - equation.advection_velocity * t
  nu = diffusivity()
  c = 0.0
  A = 1.0
  L = 2
  f = 1/L
  omega = 1.0
  scalar = c + A * sin(omega * sum(x_trans)) * exp(-nu * omega^2 * t)
  return SVector(scalar)
end
initial_condition = initial_condition_diffusive_convergence_test

# define periodic boundary conditions everywhere
boundary_conditions = boundary_condition_periodic
boundary_conditions_parabolic = boundary_condition_periodic
```
We use an indentation of 2 spaces

Reply:
OK. Is there an elegant way to commit these change? Since Michael fixed the IMEX issue, I'd proceed as follows: 1. Pull changes from upstream to my fork and branch, 2. modify IMEX branch as suggested, use IMEX timeintegration instead of SDIRK2 3. commit changes to remote. Will github then automatically change the pull request when I reload the page? Or do I use the edit button on the top right?

Reply:
Ah, sorry, I missed that with my changes, this suggestion is now outdated.

The easiest would be for you to `git pull` this branch in the local clone on your computer, then do all changes locally that you want to do, commit, and push again. I added my changes directly to your branch, so you will see them on your computer after the pull. Once you push, the PR here gets updated automagically - it always tracks the tip of your branch, so we will see all changes automatically.

Reply:
I made the suggestions above using the feature of code suggestions fro GitHub. If the weren't outdated by Michael's changes, you could just click on the "Commit suggestion" button.
![image](https://user-images.githubusercontent.com/12693098/219557057-5f7107db-0b4a-4ffc-9eda-67cb529a743a.png)